web/ui/react-app/src/Navbar.tsx

```diff
@@ -17,17 +17,18 @@ import { ThemeToggle } from './Theme';
 
 interface NavbarProps {
   consolesLink: string | null;
+  agentMode: boolean | null;
 }
 
-const Navigation: FC<NavbarProps> = ({ consolesLink }) => {
+const Navigation: FC<NavbarProps> = ({ consolesLink, agentMode }) => {
   const [isOpen, setIsOpen] = useState(false);
   const toggle = () => setIsOpen(!isOpen);
   const pathPrefix = usePathPrefix();
   return (
     <Navbar className="mb-3" dark color="dark" expand="md" fixed="top">
       <NavbarToggler onClick={toggle} className="mr-2" />
-      <Link className="pt-0 navbar-brand" to="/graph">
-        Prometheus
+      <Link className="pt-0 navbar-brand" to={agentMode ? `/agent` : `/graph`}>
```

Comment:
When you don't need to interpolate something into a string, use normal `'` single quotes instead of backticks.